analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -370,7 +370,7 @@ private class Graph(private val nodeMap: MutableMap<Identifier, Set<Identifier>>
      * of the given [root] package.
      */
     fun toPackageReferenceForest(root: Identifier): SortedSet<PackageReference> {
-        fun getPackageReference(id: Identifier, predecessorNodes: Set<Identifier> = mutableSetOf()): PackageReference {
+        fun getPackageReference(id: Identifier, predecessorNodes: Set<Identifier> = emptySet()): PackageReference {
```

Comment:
Nit: This could be squashed into the previous commit, the commit message would still fit.